ci/ray_ci/pipeline/microcheck.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Script to run microcheck on the Ray project.
+#
+# For the most parts, the steps are automatically determined, except for manually 
+# added steps such as lint
+#
+
+# set -exuo pipefail
+
+curl -fsLR -o /tmp/run_bazel "https://github.com/bazelbuild/bazelisk/releases/download/v1.16.0/bazelisk-linux-amd64"
+curl -fsL "https://raw.githubusercontent.com/ray-project/rayci/stable/run_rayci.sh" > /tmp/run_rayci.sh
+chmod +x /tmp/run_bazel
+chmod +x /tmp/run_rayci.sh
+export RAYCI_SELECT="$(/tmp/run_bazel run //ci/ray_ci/automation:determine_microcheck_step_ids),"\
+"lint-clang-format,lint-code-formatlint-untested-code-snippet,lint-banned-words,"\
+"lint-doc-readme,lint-dashboard-format,lint-copyright-format,lint-bazel-team,"\
+"lint-bazel-buildifier,lint-pytest-format,lint-test-coverage,lint-api-annotations,"\
+"lint-documentation-style,lint-all-linkcheck"
+
+/tmp/run_rayci.sh -upload
```

Comment:
wait.. this script needs to be put on the pipeline definition, not here.

having this script here will give user a way to access the buildkite access token

also, if we want to run lints, maybe give all lints a tag or something? otherwise, when people add a lint, they also have to modify the list here too.

feel that this is leaking many abstractions here and there.

Reply:
hah yes let me remove this script